forms/views/public/css/app.css

```diff
@@ -71,6 +71,11 @@
     visibility: visible;
 }
 
+.cmApp_signupContainer .cmApp_processing #cmApp_thankYouCheck img
+{
+    margin: 1px auto 1px;

```

Comment:
Instead of using margin here, could you use `display: inline` as that was the value before the change?

Reply:
Could we also change the CSS target to `.cmApp_slideOutTab #cmApp_thankYouCheck`?

Reply:
Sure thanks Yahya

Reply:
Hello @YahyaZ , I tried your suggestion but it seems that it does not change the location of the tick. Can you please give a try?

Reply:
Change pushed 👍